src/DTO/EventActivityDto.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DTO;
+
+use Symfony\Component\Validator\Constraints\DateTime;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Sequentially;
+use Symfony\Component\Validator\Constraints\Type;
+use Symfony\Component\Validator\Constraints\Uuid;
+
+class EventActivityDto
+{
+    public const CREATE = 'create';
+    public const UPDATE = 'update';
+
+    #[Sequentially([new NotBlank(), new Uuid()], groups: [self::CREATE])]
+    public mixed $id;
+
+    #[Sequentially([
+        new NotBlank(groups: [self::CREATE]),
+        new Type('string', groups: [self::CREATE, self::UPDATE]),
+        new Length(min: 2, max: 100, groups: [self::CREATE, self::UPDATE]),
+    ])]
+    public mixed $title;
+
+    #[Sequentially([
+        new Type('string', groups: [self::CREATE, self::UPDATE]),
```

Comment:
Pelo que vi no banco de dados, esse campo é um _TEXT_ e não uma _STRING_.
```suggestion
        new Type(Types::TEXT, groups: [self::CREATE, self::UPDATE]),
```